model/encodable/keys.go

```diff
@@ -21,18 +34,19 @@ func (pub NetworkPubKey) MarshalJSON() ([]byte, error) {
 	if pub.PublicKey == nil {
 		return json.Marshal(nil)
 	}
-	return json.Marshal(pub.Encode())
+	return json.Marshal(toHex(pub.Encode()))
```

Comment:
We could also call `pub.String()` but that function prefixes the value with `0x`

Reply:
yeah, didn't want to include `0x`, because golang's `hex.DecodeString` doesn't handle that case